examples/conway-game-of-life.md

```diff
@@ -2,276 +2,218 @@
 title: Conway Game Of Life
 desc: Conway Game Of Life using a central model for state management.
 date: 2021-10-26
-tags: [3rd-party, game, m.stream, m.mount, m.redraw, oninit]
+tags: [3rd-party, game, m.mount, m.redraw, oninit]
 level: expert
 version: 2.0.4
 author: boazblake
 layout: layouts/example.html
 flems:
   links:
     - ramda@0.26.1
-    - mithril-stream@2.0.0/stream.js
 ---
 
 ## JS
 
 ~~~js
-const Stream  = m.stream
 const compose = R.compose
+const flatten = R.flatten
 const range = R.range
 const without = R.without
 const values = R.values
+const prop = R.prop
+const root = document.getElementById('GameOfLife')
```

Comment:
The element doesn't exist. So either change it to `document.body` or add a minimal HTML code snippet with the ID you selected.